src/main/java/com/flippingutilities/ui/MasterPanel.java

```diff
@@ -183,56 +166,265 @@ public void mouseExited(MouseEvent e) {
 			}
 		});
 
-		JLabel discordIcon = new JLabel(Icons.DISCORD_ICON);
-		discordIcon.setToolTipText("Click to go to Flipping Utilities discord");
-		discordIcon.addMouseListener(new MouseAdapter() {
+		JLabel twitterIcon = new JLabel(Icons.TWITTER_ICON);
+		twitterIcon.setToolTipText("Click to go to Flipping Utilities twitter");
+		twitterIcon.addMouseListener(new MouseAdapter() {
```

Comment:
Looks like there is a lot of duplicated code for these icons, could we make an util for it instead?